tests/test_enterprise/views/test_handle_consent_enrollment.py

```diff
@@ -1,310 +0,0 @@
-# -*- coding: utf-8 -*-
-"""
-Tests for the ``HandleConsentEnrollment`` view of the Enterprise app.
```

Comment:
I think it might be worth adapting these tests from testing the endpoint to testing the function